### ISSUE TITLE
How can I thank you?

### ISSUE THREAD

**Issue by unrepentantgeek** [jnweiger/inkscape-silhouette#304]
If you had a donate link, I would have used it!  Is there somewhere I can donate to help fund development?

**Comment by mike-brandt** [comment]
were can i contribute 
you have saved me money 
you should get some of it

**Comment by NeighNeighNeigh** [comment]
Agreed, this implementation is so much nicer than Silhouette's software (which is such a chore to use). I would gladly contribute some $$$ 

## Update
It turns out fablabnbg has a website, as linked from their GitHub profile page (https://github.com/fablabnbg). There is a donate page available at https://www.fablab-nuernberg.de/spenden
